### ISSUE TITLE
Allow configuration of mqtt client name to allow for multiple instances

### ISSUE THREAD

**Issue by kevinanderson1** [Dielee/volvo2mqtt#93]
**Is your feature request related to a problem? Please describe.**

My family has two Volvo C40's that I would like to integrate in with Home Assistant. Ideally I would be able to run two instances (containers) of volvo2mqtt so that both C40's can be integrated in to Home Assistant. This currently fails because the volvo2mqtt instance hardcodes the mqtt client name in [src/mqtt.py](https://github.com/Dielee/volvo2mqtt/blob/main/src/mqtt.py#L26) to be `volvoAAOS2mqtt` and mqtt client names must be unique.

**Describe the solution you'd like**

Accept a configuration parameter for the mqtt client name.

**Describe alternatives you've considered**

None.

**Additional context**

I've manually built a second container with the name hardcoded to something else and have been running it without any issues.

**Comment by Dielee** [comment]
Are your C40s running with different Volvo accounts?

**Comment by kevinanderson1** [comment]
Yes each car is tied to a different Volvo account.

**Comment by Dielee** [comment]
Okay. I will implement an auto generating unique ID for each instance. So there will be no user input required.

**Comment by kevinanderson1** [comment]
Thank you! I took a quick attempt at implementing a user configurable
option in a PR but I can close that out later today.

**Comment by Dielee** [comment]
Yes, thanks a lot, but the configuration is already a mess. I try to avoid any addition...

**Comment by Dielee** [comment]
One more question: If you are using different volvo accounts, your login mail differs, right ?

**Comment by Dielee** [comment]
You are not using the HA Addon, right ?

**Comment by kevinanderson1** [comment]
No I am not. I have the instances deployed in containers and managed under
systemd units.

**Comment by Dielee** [comment]
v1.8.6 is building :)

**Comment by kevinanderson1** [comment]
Thank you! I'll update this evening and report back.

**Comment by Dielee** [comment]
Build is done. Feel free to test later today!
